clients/urls.py

```diff
@@ -1,12 +1,11 @@
 from django.urls import path
-from clients.views import ClientsView,ClientViewView,ClientAddView, ClientEditView,ClientDeleteView
+from clients import views
```

Comment:
explicit is better than implicit

from clients.views import (
    ClientListView, 
    ClientView,
)